### PR TITLE
qa fixes

### PR DESCRIPTION
**Pull request by hthillman**
updates copy in a few places

prepopulates search results with featured pipelines

### CODE REVIEW

File: apps/app/components/header/fetchPipelines.tsx [hunk 1]
```diff
@@ -21,3 +21,25 @@ export async function fetchPipelines(query: string) {
...
+
+
+export async function fetchFeaturedPipelines(limit: number = 3) {
+  const supabase = await createServerClient();
+  try {
+    const { data, error } = await supabase
+      .from("pipelines")
+      .select("*")
+      .eq("is_featured", true)
+      .limit(limit);
```
Comment (victorges) [inline apps/app/components/header/fetchPipelines.tsx:33]: Does this run on the frontend? Does that mean our tables are public? Might be a question for @suhailkakar

File: apps/app/components/playground/output.tsx [hunk 2]
```diff
@@ -65,17 +65,21 @@ export default function Output({
...
+        <div className="text-muted-foreground">
+              Your stream may take up to 90 seconds to start.
+        </div>
         {streamInfo?.output_stream_url && (
           <Button variant="outline" onClick={() => copyLogs()}>
             <Copy className="mr-2" /> Copy logs
           </Button>
         )}
-        <Button
+        {/* COMMENTED OUT UNTIL WE HAVE TIME TO IMPLEMENT FETCHING PIPELINE DETAILS FROM THE DB
```
Comment (victorges) [inline apps/app/components/playground/output.tsx:76]: Might be easier to find to add it as a `TODO:` format comment (just having it a prefix to the comment then what needs to be done)

File: apps/app/components/playground/try.tsx [hunk 3]
```diff
@@ -326,7 +326,7 @@ export default function Try({
               <BroadcastWithControls ingestUrl={streamUrl} />
             ) : (
               <p className="text-muted-foreground">
-                Your stream is loading. This may take up to 1 minute, so please be patient.
+                Waiting for webcam to start...
```
Comment (victorges) [inline apps/app/components/playground/try.tsx:329]: Isn't this possibly misleading? Sounds like a "user fault" message 🤔

**Review by victorges — APPROVED** [review]
LGTM either way

File: apps/app/components/header/fetchPipelines.tsx (hunk 1 above)
Comment (hthillman) [reply apps/app/components/header/fetchPipelines.tsx:33]: Yeah, this is a question for @suhailkakar. I thiiiink it's just the pipelines table

This is copied from similar code that handles pipeline searching

File: apps/app/components/playground/output.tsx (hunk 2 above)
Comment (hthillman) [reply apps/app/components/playground/output.tsx:76]: truuuu

File: apps/app/components/playground/try.tsx (hunk 3 above)
Comment (hthillman) [reply apps/app/components/playground/try.tsx:329]: this is on the broadcast component - I moved the "please wait" message to another spot

**Pull request merged**